<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1589</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pbCreateNewTest">
         <property name="text">
          <string>Create new test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Test number</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leTestNumber"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pbGripBall">
         <property name="text">
          <string>Grip ball</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Target detected at</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="leTargetDetected"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ball detected at</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ball landed at</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Velocity af toss</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="leVelocityToss"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Precision of toss</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="leTossPrecision"/>
       </item>
       <item row="19" column="0">
        <widget class="QPushButton" name="pbAddToss">
         <property name="text">
          <string>Add toss to database</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of tosses in test</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QLineEdit" name="leNumberOfTosses"/>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Average precision in test</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QLineEdit" name="lePrecision"/>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="leBallDeteced">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="leBallLanded"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pbDetectTarget">
         <property name="text">
          <string>Detect target</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pbTossBall">
         <property name="text">
          <string>Toss ball</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="display_video">
         <property name="text">
          <string>display_video</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1589</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
